.github/ISSUE_TEMPLATES/Feature Request.yml

```diff
@@ -0,0 +1,28 @@
+name: Feature Request
+description: Request a new feature for nesta_ds_utils
+title: "[Feature]: "
+labels: ["enhancement"]
+body:
+  - type: checkboxes
+    id: slack
+    attributes:
+      label: Slack Post
+      description: By submitting this issue, you acknowledge that you have suggested this feature in Slack and it has been approved
```

Comment:
If using Slack then maybe good to nominate a channel and say which one here. Also not sure who is in charge of approving - maybe 'agreed' is better?

I also wonder if there's a case for putting the feature issue in GitHub first, then telling people on Slack. Keeping a track of the features requested on here is useful for future contributors/requesters, because they can search here for things that have been requested previously and approved or denied.

Don't feel massively strongly about it either way though.